views/partials/banners/sign_up_banner.hbs

```diff
@@ -1,16 +1,22 @@
-<div id="sign-up-banner" class="banner">
-  <div class="section-wrapper">
-    <div class="fourth subscribe-icon-wrapper">
-      {{> sprite symID="subscribe"}}
-    </div>
-    <div class="{{#unless breachIsSensitive }} half {{/unless}} {{#if breachIsSensitive }} three-fourths {{/if}}">
-      <h2 class="medium secondary-title">{{{fluentFormat req.supportedLocales "signup-banner-headline"}}}</h2>
-      <p class="demi">{{{fluentFormat req.supportedLocales "signup-banner-blurb"}}}</p>
-    </div>
-    {{#unless breachIsSensitive}}
-      <div class="fourth">
-        {{> sign_up_button}}
+{{#unless req.session.user}}
+  {{#if FXA_ENABLED}}
+    {{> fxa_enabled/fxa_sign_up_banner}}
+  {{else}}
+    <div id="sign-up-banner" class="banner">
+      <div class="section-wrapper">
+        <div class="fourth subscribe-icon-wrapper">
+          {{> sprite symID="subscribe"}}
+        </div>
+        <div class="{{#unless breachIsSensitive }} half {{/unless}} {{#if breachIsSensitive }} three-fourths {{/if}}">
```

Comment:
I'm not the best w/ mustache conditionals, but not sure if we could replace the `#unless` with an `else`, or if I'm missing something:

```html
<div class="{{#if breachIsSensitive }} three-fourths {{else}} half {{/if}}">
```